packages/tools/src/BoltTool.ts

```diff
@@ -18,9 +18,9 @@ export const BoltTool: Tool = {
 
   async isMonorepoRoot(directory: string): Promise<boolean> {
     try {
-      const pkgJson = (await fs.readJson(
+      const pkgJson = JSON.parse((await fs.promises.readFile(
```

Comment:
is `fs.promises` supported by node 14? this is our declared supported version in package.json. Does this version support `"fs/promises"`? I prefer the latter over `fs.promises`

Reply:
added in node 10: https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fspromises_readfile_path_options